proposals/20200909-fargate-falco-support.md

```diff
@@ -0,0 +1,156 @@
+# Falco userspace launch support
+
+## Summary
+
+This is a proposal on how to adress deployment of falco in a userspace
+ environment.
+
+There has been a rise in popularity of serverless architectures that are 
+managed and make kernel instrumentation impossible. This makes it quite hard to
+monitor the runtimes. This project is about defining a patching language/definition
+that allows falco to be deployed alongside a container in userspace.
+
```

Comment:
* Yes. This is actually a project that introduces a definition on how to patch containers, and several runtimes that apply that definition.
* Following the definitions in the repo - this will be in the sandbox status unless further interest is demonstrated.

I am currently doing a pass of the proposal to make this clearer.